src/tests/functional/plugin/gpu/shared_tests_instances/behavior/ov_infer_request/perf_counters.cpp

```diff
@@ -34,11 +34,18 @@ auto configs = []() {
 };
 
 auto Multiconfigs = []() {
-    return std::vector<ov::AnyMap>{{ov::device::priorities(CommonTestUtils::DEVICE_GPU)}};
+    return std::vector<ov::AnyMap>{{ov::device::priorities(CommonTestUtils::DEVICE_GPU)},
+                                   {ov::device::priorities(CommonTestUtils::DEVICE_GPU, CommonTestUtils::DEVICE_CPU),
```

Comment:
generally, we are using macro ENABLE_INTEL_CPU to make sure tests need CPU will be scheduled to suitable hardware nodes.

Reply:
updated